projects/packages/woocommerce-analytics/src/class-woo-analytics-trait.php

```diff
@@ -410,7 +435,12 @@ public function process_event_properties( $event_name, $properties = array(), $p
 			)
 		);
 
-		$js = "{'_en': '" . esc_js( $event_name ) . "'";
+		// ch param is needed to identify ClickHouse queries in the JS Analytics library. Remove prefix also for ClickHouse
+		if ( $clickhouse ) {
+			$js = "{'_en': '" . esc_js( str_replace( 'woocommerceanalytics_', '', $event_name ) ) . "','ch':'1'";
```

Comment:
Why is the prefix removed for ClickHouse?

Reply:
Because all the events are populated in the ClickHouse table named `woocommerce_analytics_store_events` that is exclusive for WooCommerce Analytics. 

Since we found it redundant we decided to have them without prefix. Notice this is different that the Nosara tracks which includes more than just the `woocommerceanalytics_` events.

Do you suggest to keep it?

Reply:
I keep the prefix here as well. So we can keep it homogenous with Nosara Tracks https://github.com/Automattic/jetpack/pull/41984/commits/fef8e05c4734e3117036d8978710cb3afcd16446